js/app.js

```diff
@@ -0,0 +1,100 @@
+//What is the difference between interpolation and concatenation?
+// Interpolation uses a variable to represent a value.
+//Concatenation combines two 'strings' together.
+
+//Give an example of each.
+// Interpolation:
+//// const verb = 'believe'
+// const noun = 'cheese'
+// console.log("Please " + verb + " the " + noun + ".")
+//
+//Concatenation:
+//const cityName = 'Astoria'
+//const weather = 'rainy'
+//console.log(cityName + " is " + weather + " this time of year.");
+//**ARE ALL CONCATENATIONS INTERPOLATIONS BUT NOT ALL INTERPOLATIONS ARE CONCATENATIONS??**
+
+//What does the acronym DRY stand for?
+//Don't Repeat Yourself
+
+//Why should we pay attention to it?
+//Helps utilize best practices in programming. DRY makes our code easier to read by other programmers
+//and helps us avoid bulky code and unessecary repetition.
+
+//What programming tools have we learned to write DRY code?
+//Thematically we discussed understaning the specific task at hand before rendering the code.
+//Specifially, we learned to use loops to avoid code repitition.
+
+//Section 2: Boolean Expressions
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a<b);
+console.log(c>d);
+console.log('Name'=== 'Name');
+console.log(a<b<c);
+console.log(a<a<d);
+console.log(e==='Kevin');
+console.log(48!=='48');
+
+
+//Section 3: While loops
+//while (true) {
+//console.log('Do not run this loop');
+//}
+//This loop is an infinite loop. The loop does not contain a "code block" and
+//there is no control expression.
+
+//Section 3: Inifinite Loop II
+//Is this loop inifinite? Why or why not?
+//const runProgram = true;
+// while (runProgram) {
+	// console.log('Do not run this loop');
+	// runProgram = false;
+// }
+//This loop is not infinite. It meets the while loop criteria using:
```

Comment:
i don't understand what you mean "meets the while loop criteria"

Reply:
but you are correct that the loop is not infinite.  Where we say "runProgram" = false we stop the loop.  We think we're stopping it because we're making the condition that keeps it running stop being true, but actually we stop it because trying to change the value of a `const` will throw an error